packages/nitro-protocol/config/jest/jest.contracts.config.js

```diff
@@ -3,4 +3,5 @@ config.testMatch = ['<rootDir>/test/contracts/**/*.test.ts'];
 config.reporters = ['default'];
 config.globalSetup = '<rootDir>/jest/contract-test-setup.ts';
 config.globalTeardown = '<rootDir>/jest/contract-test-teardown.ts';
+config.testTimeout = 90_000;
```

Comment:
I can't think of a reason that these changes should require, on average, substantially more time in the test suites. Any comment?

Reply:
I also cannot think of a compelling reason 😞 . I think we may have been quite close to the existing timeout already, and we have disturbed the number of tests that run (concurrently), which could lead to hard-to-predict changes in test runtime. 

I didn't yet try and reduce this timeout again, there is probably quite a bit of slack in it now.